lib/Service/NotificationTrackerService.php

```diff
@@ -57,7 +57,9 @@ private function handleException(\Exception $e) {
 	}
 
 	/**
-	 * @throws NotFoundException
+	 * @param string $userId
+	 *
+	 * @return NotificationTracker
```

Comment:
If this method don't throw an exception, would be good to remove all the docblock because haven't a custom description about method, arguments or return and this method have the argument and the return strong typed.

Reply:
Given that we cannot use typed parameters and return codes yet due to PHP 7.3 support I kept it for now. I agree that it should go once we switch to a newer PHP version with typing support.